.sequelizerc

```diff
@@ -1,13 +1,8 @@
-
-
-
-
-
 require('@babel/register')
  const path = require('path');
 module.exports = {
   "config": path.resolve('./src/database', 'config.js'),
-  "models-path": path.resolve('./src/models'),
+  "models-path": path.resolve('./src/modules/user/model'),
```

Comment:
Yes. He also need to add the `babel transform runtime plugin` again for heroku.

Reply:
Each module should have it own model (user.model, accommodation model ...) but their will be a central index.js to link the relationships between each models. Normally all models ought to be inside model folder because when you run sequelize init it auto generate all the folders paths specified in the sequelizerc